docs/src/components/pricing/PricingTable.tsx

```diff
@@ -813,6 +911,33 @@ const premiumData: Record<string, React.ReactNode> = {
   'data-grid/localization': yes,
   'date-picker/simple': yes,
   'date-picker/range': yes,
+
+  // -- charts - components --
+  'charts/line': yes,
+  'charts/bar': yes,
+  'charts/scatter': yes,
+  'charts/pie': yes,
+  'charts/sparkline': yes,
+  'charts/gauge': pending,
+  'charts/treemap': pending,
+  'charts/radar': pending,
+  'charts/funnel': pending,
+  'charts/sankey': pending,
+  'charts/gantt': pending,
+  'charts/gantt-advanced': pending,
```

Comment:
To avoid creating the commitment to building these features in the short-medium term, we can change from "pending" to "TBD". 

If possible, bellow the "TBD" text we could have something like "**Interested?**" as a link.
The link sends the user to a Google form where they can confirm and expand on their expectations for these premium features. 
We could also consider having the TBD text itself as a link.

```suggestion
  'charts/gantt-advanced': TBD,
```

Wdyt?

Reply:
Why using a Google form instead of a github issue?

Reply:
I believe people would be more compelled to share their use cases, but I suppose we can also create GitHub issue and make the questions pop up on the description. It's also a good thing that we can redirect each "TBD" to a different issue.